packages/cli/__tests__/zosjobs/__integration__/submit/data-set/__snapshots__/cli.zos-jobs.submit.data-set.integration.test.ts.snap

```diff
@@ -61,6 +61,14 @@ exports[`zos-jobs submit data-set command should display the help 1`] = `
 
       A file extension to save the job output with. Default is '.txt'.
 
+   --jcl-symbols  | --js (string)
```

Comment:
Integration test snapshots only changed for `jobs submit data-set` and `jobs submit stdin`. Should a test be added that checks the CLI help text of the `jobs submit local-file` command? I don't think we necessarily need to add such a test in this PR, just bringing it up since I noticed this.

Reply:
I added a help test for jobs submit local-file.